examples/code/imgui_example/imgui_example.cpp

```diff
@@ -24,8 +24,8 @@ namespace pen
     pen_creation_params pen_entry(int argc, char** argv)
     {
         pen::pen_creation_params p;
-        p.window_width = 1280;
-        p.window_height = 720;
+        p.window_width = 1280.0f;
```

Comment:
The .0f isn't necessary anymore since the window_width and height should be u32 again?

Reply:
I did change  this let me recheck.